src/action_types/groups.ts

```diff
@@ -12,7 +12,7 @@ export default keyMirror({
     RECEIVED_GROUP_TEAMS: null,
     RECEIVED_GROUP_CHANNELS: null,
 
-    RECEIVED_GROUP_MEMBERS: null,
```

Comment:
I take it nobody is listening for `RECEIVED_GROUP_MEMBERS` this anymore? Neither web nor mobile?

Reply:
yup